server/server.go

```diff
@@ -51,7 +54,7 @@ func NewServer(
 	reviewTimeout time.Duration,
 	pushTimeout time.Duration,
 ) *Server {
-	return &Server{p, fileGetter, analyzers, eventOp, commentOp, reviewTimeout, pushTimeout}
+	return &Server{false, p, fileGetter, analyzers, eventOp, commentOp, reviewTimeout, pushTimeout}
```

Comment:
I'm late in the discussion... but... I'd prefer not to use positional arguments.

Reply:
sorry David but it's unrelated to this PR. It uses the same style that was used before. We can fix it in another PR later if you want.